internal/datacoord/handler.go

```diff
@@ -263,6 +272,124 @@ func (h *ServerHandler) GetQueryVChanPositions(channel RWChannel, partitionIDs .
 	}
 }
 
+func (h *ServerHandler) GetCurrentSegmentsView(ctx context.Context, channel RWChannel, partitionIDs ...UniqueID) *SegmentsView {
```

Comment:
Use `GetQueryVChanPositions` to prevent duplicated codes.

Reply:
It's not exactly the same as `GetQueryVChanPositions`, I have extracted the common functions.